iothub_client/tests/iothubtr_amqp_tel_msgr_ut/iothubtr_amqp_tel_msgr_ut.c

```diff
@@ -1665,11 +1665,109 @@ static TELEMETRY_MESSENGER_HANDLE create_and_start_messenger2(TELEMETRY_MESSENGE
     return handle;
 }
 
+char* umock_stringify_BINARY_DATA()
+{
+    char* result = (char*)TEST_malloc(1);
+    result[0] = '\0';
+    return result;
+}
+
+int umock_are_equal_BINARY_DATA()
+{
```

Comment:
Shouldn't it take argumetns and compare?

Reply:
same as above